app/models/linkset.rb

```diff
@@ -11,11 +11,21 @@ class Linkset < ApplicationRecord
       message: "does not appear to be a valid URL"
```

Comment:
for each link, should also reset the verified_at field to nil if the link changes